### PR TITLE
[FE-8] Categories points bars

### PR DESCRIPTION
**Pull request by AnCichocka**
- points bar added
- deleted lvl point bar from student card for now

image

### CODE REVIEW

**Comment by AnCichocka** [comment]
Front should works just fine for categories progress bars. I encounter a problem. Here is example response:
 ```{
    "data": {
        "getSumOfPointsForStudentByCategory": [
            {
                "maxPoints": 140.0,
                "sumOfAll": 0.0,
                "sumOfBonuses": 0.0,
                "sumOfPurePoints": 0.0,
                "category": {
                    "categoryName": "LABORATORY",
                    "categoryId": 1,
                    "label": "",
                    "__typename": "CategoriesType"
                },
                "__typename": "CategoryPointsSumType"
            },
            {
                "maxPoints": 70.0,
                "sumOfAll": 12.0,
                "sumOfBonuses": 0.0,
                "sumOfPurePoints": 12.0,
                "category": {
                    "categoryName": "TEST",
                    "categoryId": 2,
                    "label": "",
                    "__typename": "CategoriesType"
                },
                "__typename": "CategoryPointsSumType"
            },
            {
                "maxPoints": 150.0,
                "sumOfAll": 190.0,
                "sumOfBonuses": 190.0,
                "sumOfPurePoints": 0.0,
                "category": {
                    "categoryName": "PROJECT",
                    "categoryId": 3,
                    "label": "",
                    "__typename": "CategoriesType"
                },
                "__typename": "CategoryPointsSumType"
            },
            {
                "maxPoints": 130.0,
                "sumOfAll": 0.0,
                "sumOfBonuses": 0.0,
                "sumOfPurePoints": 0.0,
                "category": {
                    "categoryName": "EVENT",
                    "categoryId": 4,
                    "label": "",
                    "__typename": "CategoriesType"
                },
                "__typename": "CategoryPointsSumType"
            }
        ]
    }
}```

- Why some categories (for example) project is returned only as bonuses? 
- Why event is returned as category here? I think we agreed that event can only return bonuses, not points itself - so maybe it shouldn't have its own points bar

File: frontend/src/components/PointsBar.tsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
+import { Styles } from "../utils/Styles";
+
+const styles: Styles = {
+  container: {
+    display: "flex",
+    flexDirection: "column",
+    gap: 4,
+  },
+  empty: {
+    height: 24,
```
Comment (Szyntos) [inline frontend/src/components/PointsBar.tsx:10]: why not 25

it's +1, so a small change, and it will be a prettier number

image

Comment (Szyntos) [reply frontend/src/components/PointsBar.tsx:10]: joke ofc

**Review by Szyntos — APPROVED** [review]
I don't have any questions, looks good <3

File: frontend/src/components/PointsBar.tsx (hunk 1 above)
Comment (AnCichocka) [reply frontend/src/components/PointsBar.tsx:10]: In web design (and design overall) numbers are 4*x or 8*x in general, it scales better

**Pull request merged**